homeassistant/components/elkm1/climate.py

```diff
@@ -106,90 +112,90 @@ def target_temperature(self):
         return None
 
     @property
-    def target_temperature_high(self):
+    def target_temperature_high(self) -> float | None:
         """Return the high target temperature."""
         return self._element.cool_setpoint
 
     @property
-    def target_temperature_low(self):
+    def target_temperature_low(self) -> float | None:
         """Return the low target temperature."""
         return self._element.heat_setpoint
 
     @property
-    def target_temperature_step(self):
+    def target_temperature_step(self) -> float:
         """Return the supported step of target temperature."""
         return 1
 
     @property
-    def current_humidity(self):
+    def current_humidity(self) -> int | None:
         """Return the current humidity."""
         return self._element.humidity
 
     @property
-    def hvac_mode(self):
+    def hvac_mode(self) -> str | None :
```

Comment:
I think we are stuck with always returning OFF for the unknown case since it’s typed to be str only